_data/landscape/meshes.yml

```diff
@@ -108,6 +121,8 @@
   tracing: "OpenTracing compatible"
   encryption: "Yes"
   multi-tenant-score: 1
+  spiffe: "?"
```

Comment:
Consul has partial SPIFFE support, but doesn't support the Workload API or Federation, and thus is not interoperable with other SPIFFE implementations.

Reply:
This is Grey Matter.